src/types/entities/ui.ts

```diff
@@ -103,8 +103,8 @@ export default class Ui {
 			left: 0,
 			width: '55%', // blocksBox ? '55%' : '75%',
 			height: '100%',
-			content:
-				'{left}error and trace\noutput will appear here\nand scroll down{/left}',
+			content: `{left}\n${usage}{/left}`,
+			//			`{left}error and trace\noutput will appear here\nand scroll down{/left}\n${usage}`,
```

Comment:
Do you think this bit of information should be permanently removed from the initial box population?

Reply:
I agree, and think that this issue is more broadly covered by #3